projects/vdk-plugins/vdk-jupyter/convert-job-tutorial-temp/tutorial.md

```diff
@@ -0,0 +1,139 @@
+Learn how to convert a VDK Job with .sql and .py steps into a Job with notebook steps using VDK Jupyter Notebook UI.
+
+Time Commitment: About 5-10 minutes.
+
+## Prerequisites
+
+Before you start with the conversion, please ensure you have the following:
+
+<details>
+  <summary>Installed Software and Services</summary>
```

Comment:
The sumamry are not very noticeable. 
make them bold or even better into a heading like that 

```suggestion
  <summary><h3>Developed Data Job</h3></summary>
```